utils/queryApiGateway.tsx

```diff
@@ -48,6 +54,11 @@ export function getApiVars(): ApiEnvVars {
   const pfxFilename: string = process.env.P12_FILE ?? ''
   apiEnvVars.pfxPath = path.join(certDir, pfxFilename)
 
+  // Some certificates have an import password
+  if (process.env.PFX_PASSPHRASE) {
```

Comment:
Could this if() be omitted, and just proceed to the line below, for the same result? And if it's possible for process.env.PFX_PASSPHRASE to be an empty string, maybe the line below could use || instead of ??